goalc/emitter/Register.h

```diff
@@ -96,6 +96,13 @@ class Register {
 
   std::string print() const;
 
+  /*
+          Our XMM Registers are 4 packed single-precision floating points
+    In the order (from left->right a.k.a most significant to least significant):
+    W | Z | Y | X
+          */
+  enum class XMM_ELEMENT { X, Y, Z, W, NONE };
```

Comment:
Can we rename this to `VF_Element`? The floating point numbers also go in `xmm`s so I try to use `vf` when referring to an `xmm` that has a vector.